webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/servers/PolicyServerManagementService.scala

```diff
@@ -316,16 +323,31 @@ class PolicyServerManagementServiceImpl(
    */
   val lock = Semaphore.make(1).runNow
 
+
+
+  private def getDefaultSettingEntryIfMissing = {
+    val entry = dit.APPCONFIG.propertyModel(RudderWebPropertyName(PROP_NAME))
+    entry.resetValuesTo(RudderLDAPConstants.A_PROPERTY_VALUE, """[{"id":"root","allowed-networks":[]}]""")
+    entry
+  }
+
   private def getLdap(con: RwLDAPConnection) = {
     for {
-      entry   <- con.get(dit.APPCONFIG.propertyDN(RudderWebPropertyName(PROP_NAME))).notOptional(s"LDAP setting entry ${PROP_NAME} was not found. It may be a bug, please report it.")
```

Comment:
we need to be able to be missing allowed network setting at the begining of migration from 6.2